flink-runtime/src/main/java/org/apache/flink/runtime/state/RegisteredBroadcastBackendStateMetaInfo.java

```diff
@@ -52,6 +52,23 @@ public RegisteredBroadcastBackendStateMetaInfo(
 		this.valueSerializer = Preconditions.checkNotNull(valueSerializer);
 	}
 
+	public RegisteredBroadcastBackendStateMetaInfo(RegisteredBroadcastBackendStateMetaInfo copy) {
```

Comment:
The code should not work with raw types when we can avoid it and use `RegisteredOperatorBackendStateMetaInfo<S> copy` as parameter.

Reply:
👍